homeassistant/components/vesync/fan.py

```diff
@@ -124,6 +126,9 @@ def extra_state_attributes(self):
         if hasattr(self.smartfan, "night_light"):
             attr["night_light"] = self.smartfan.night_light
 
+        if hasattr(self.smartfan, "display_state"):
+            attr["display_state"] = self.smartfan.display_state
+
```

Comment:
Shouldn't this be a separate sensor?

Reply:
Currently, this integration does not have any sensors exposed.  When adding it to home assistant the only thing that is there is 1 ‘entity’ which is the main fan. I’d love to be able to expose the main sensors but fear it’s above my skill level. Lines 129-131 add to the already existing attributes. I can remove this line from this PR if needed. 

For reference, here are the state attributes for this entity.  

```
speed_list:
  - 'off'
  - low
  - medium
  - high
  - auto
  - sleep
preset_modes:
  - auto
  - sleep
speed: low
percentage: 33
percentage_step: 33.333333333333336
preset_mode: null
child_lock: false
night_light: 'off'
air_quality: 6
mode: manual
filter_life: 60
display_state: false
friendly_name: Bedroom Purifier
supported_features: 1 
```

Reply:
> Currently, this integration does not have any sensors exposed.

Then we should add that. We should not introduce more items of a pattern we actually no longer accept in Home Assistant.

> I’d love to be able to expose the main sensors but fear it’s above my skill level. Lines 129-131 add to the already existing attributes. I can remove this line from this PR if needed.

In that case, removing the new attribute sounds like an option.

Reply:
Thank you. I have removed that from the file - hoping it has filtered through to this request. Please let me know if there is anything else. I will begin researching adding sensors and hope to maybe add that via another PR later this year.